src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -183,22 +182,22 @@ async def set_pi_constants(self, prop_const, int_const):
         :param prop_const: Value of the proportional term in range [0; 255].
         :param int_const: Value of the integral term in range [0; 255].
         """
-        payload = st.pack("<HHH", 1, prop_const, int_const)
+        payload = st.pack("<HHH", _CHANNEL, prop_const, int_const)
```

Comment:
```suggestion
        payload = st.pack("<HHH", Tpz._CHANNEL, prop_const, int_const)
```
Use class constants.